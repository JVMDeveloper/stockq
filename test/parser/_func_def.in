def int myfunc (int x, int y) {
    3+4;
}
