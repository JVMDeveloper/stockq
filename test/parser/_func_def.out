def int myfunc (x, y) {
	Binop(IntLiteral(3), Add, IntLiteral(4))
}

